test api/REST Client/Cloud Migration API/Cloud Migration company.http

```diff
@@ -0,0 +1,61 @@
+@clientid = 7e930f8e-fac4-43a6-a242-3a4079ac4553
+@clientsecret = clientSecret
+@scope = https://api.businesscentral.dynamics.com/.default
+@tenant = ffb102fa-38b1-4d6b-ad59-1192df615c1e
+@environmentName = Sandbox1
+@aPIPublisher = ciellos
```

Comment:
Also have you created this API Pages with ciellos publisher ? I was unable to find them in this Pull Request and in current solution..

Reply:
Code reviewed. It also has no use in the current solution so I decided to remove it.